### ISSUE TITLE
Donation modal opened from within loop has excess padding

### ISSUE THREAD

**Issue by ericnicolaas** [Charitable/Reach#49]
We have this:

```
#charitable-donation-form-modal #charitable-donation-form {
    padding: 0;
}
```
We need it for donation modals in the loop too: 
```
#charitable-donation-form-modal #charitable-donation-form,
#charitable-donation-form-modal-loop #charitable-donation-form {
    padding: 0;
}
```

This wouldn't be an issue for most people but it is if you use a code snippet to add the Donate button to the campaigns in the loop.

**Comment by ericnicolaas** [comment]
To see the actual issue, you need to use this code snippet:

https://github.com/Charitable/library/blob/master/themes/reach/add-donate-button-to-campaign-loop.php

**Comment by ericnicolaas** [comment]
Closed by 820ef58
